### ISSUE TITLE
labels are not showing inside CoordinatorLayout

### ISSUE THREAD

**Issue by nandhu** [Clans/FloatingActionButton#450]
I am using fab inside CoordinatorLayout. All icons are showing. But labels are not showing.

Thanks

**Comment by imranbaigshortcut** [comment]
use this **app**:fab_label="test label"
instead of  **fab**:fab_label="test label"

**Comment by Mapk26** [comment]
> use this **app**:fab_label="test label"
> instead of **fab**:fab_label="test label"

Yes with **app:**  it's working nice, it is needed also for app:fab_size="mini" 
In my case I'm using a RelativeLayout

**Comment by arams** [comment]
Thanks a lot... wasted couple of hours before seeing this :(
